### ISSUE TITLE
"invalid syntax" on typing.NamedTuple

### ISSUE THREAD

**Issue by Tenzer** [PyCQA/pylint#2745]
### Steps to reproduce
1. Place the following content in a file:
    ```py
    from typing import NamedTuple

    Bar = NamedTuple("bar", [])

    Bar()
    ```
2. Run Pylint on it

### Current behavior
Pylint print out an "invalid syntax" error:

    $ pylint test.py
    internal error with sending report for module ['test.py']
    Parsing Python code failed:
    invalid syntax (<unknown>, line 1)

### Expected behavior
The "invalid syntax" error wouldn't happen and the file would be checked normally.

### pylint --version output

    $ pylint --version
    pylint 2.3.0-dev1
    astroid 2.2.0-dev
    Python 3.7.2 (default, Feb 12 2019, 08:15:36)

I've also tried it with the stable versions of Pylint and Astroid, and the same error happens.

---

As far as I can tell this happens due to the second attribute to `NamedTuple()` being an empty list. If that attribute either is removed or changed to for instance `["baz"]` the file is checked like normal.

The code in the example does however execute fine in Python, and I've taken it out of a bigger application where I saw the "invalid syntax" error reported from Pylint, and managed to boil it down to this minimal example.

**Comment by Tenzer** [comment]
@PCManticore Thanks! That was an impressively quick turnaround :)

**Comment by PCManticore** [comment]
Thanks for reporting the issue! :)
